hbase-shell/src/test/java/org/apache/hadoop/hbase/client/TestShell.java

```diff
@@ -35,7 +35,7 @@
   @Test
   public void testRunShellTests() throws IOException {
     System.setProperty("shell.test.exclude", "replication_admin_test.rb,rsgroup_shell_test.rb," +
-      "admin_test.rb,table_test.rb,quotas_test.rb");
+      "admin_test.rb,table_test.rb,quotas_test.rb,admin2_test.rb");
```

Comment:
What's the story of `admin2_test.rb`, why is it related to this fix?

Reply:
It's not related to this fix. But admin2_test.rb is already tested in TestAdminShell2, TestShell can skip it rather than test it twice?